src/main/java/model/Basket.java

```diff
@@ -1,12 +1,30 @@
 package model;
 
+import javax.persistence.*;
 import java.util.List;
 
+@Entity
+@Table(name = "baskets")
 public class Basket {
 
-    Long id;
-    User user;
-    List<Product> productList;
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @OneToOne(cascade = CascadeType.MERGE)
+    private User user;
+
+    @ManyToMany(cascade = CascadeType.MERGE , fetch = FetchType.EAGER)
+    @JoinTable(
+            name = "baskets_products",
+            joinColumns = { @JoinColumn(name = "basket_id") },
+            inverseJoinColumns = { @JoinColumn(name = "product_id") }
+    )
+    private List<Product> productList;
```

Comment:
```suggestion
    private List<Product> products;
```
better doesn't add word 'list' to the name of List<> (the same about name of the interface, methods and classes)
bad practice:
public interface DaoInterface{
}
public void addElementMethod(T t){
}
private List<Product> productList